File: examples/extending-objects/Inheritance/Java/Employee.java
public class Employee extends Person {
  private int staffNumber;
  
  public Employee(String firstName, String lastName, int age, int staffNumber) {
    super(firstName, lastName, age);
    this.staffNumber = staffNumber;
  }
  
  public String toString() {
    return super.toString() + ", " + this.staffNumber;
  }
}

File: examples/extending-objects/Inheritance/C++/Employee.cpp
#include <string>
using namespace std;
#include "Person.cpp"

class Employee : public Person {
  private:
	 int staffNumber;
  
  public:
	 Employee(string firstName, string lastName, int age, int staffNumber) : Person(firstName, lastName, age) {
		this->staffNumber = staffNumber;
	}
  
	 string toString() {
		return Person::toString() + ", " + to_string(this->staffNumber);
	}
};